.github/workflows/main.yml

```diff
@@ -172,6 +172,10 @@ jobs:
         yum install -y https://packages.endpointdev.com/rhel/7/os/x86_64/endpoint-repo.x86_64.rpm
         yum install -y git wget
 
+    - name: Install IPC-Cmd for OpenSSL
```

Comment:
Kindly if this is not a code checkout dependency then add it to [install_centos_dependencies_build.sh](https://github.com/os-fpga/FOEDAG/blob/main/.github/workflows/install_centos_dependencies_build.sh)

Reply:
Thanks Nadeem. 

But maybe in next PR. 

I had retried dozens of triggering of CI run - empty commit and dummy change - and now finally all the flows are passing. Otherwise it keep giving error

![image](https://github.com/os-fpga/FOEDAG/assets/124318336/d98dddb7-6e71-42e7-bc24-ba8a33e60a7b)

If other than this, you do not have any more feedback, please approve this, so I can move-on to FOEDAG_rs.